com.github.diamondburned.gtkcord4.yml

```diff
@@ -0,0 +1,40 @@
+id: com.github.diamondburned.gtkcord4
+runtime: org.gnome.Platform
+runtime-version: '43'
+sdk: org.gnome.Sdk
+command: gtkcord4
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --filesystem=xdg-run/keyring
+  - --device=dri
+
+modules:
+  - name: gtkcord4
+    sources:
+      - type: file
+        path: com.github.diamondburned.gtkcord4.desktop
+      - type: file
+        path: com.github.diamondburned.gtkcord4.svg
+      - type: file
+        path: com.github.diamondburned.gtkcord4.metainfo.xml
+    buildsystem: simple
+    build-options:
+      build-args:
+        - --share=network
```

Comment:
This is not allowed on Flathub. The Internet connection is blocked during the build. All Builds must be done locally.

Reply:
https://github.com/flatpak/flatpak-builder-tools/tree/master/go-get can be used to generate required files.